tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php

```diff
@@ -669,6 +671,100 @@ public function testRegistrationOtherObjects(
+
+        $externalCredentials['connection']['backend'] = 'bigquery';
+        $externalCredentials['connection']['credentials'] = $this->getCredentialsArray();
+        $externalCredentials['connection']['schema'] = sha1($description) . '_external_bucket';
+
+        $db = WorkspaceBackendFactory::createWorkspaceBackend($externalCredentials);
+        $db->createTable('TEST', [
+            'AMOUNT' => 'INT',
+            'DESCRIPTION' => 'STRING',
+        ]);
+
+        // register external bucket
+        $idOfBucket = $testClient->registerBucket(
+            $testBucketName,
+            $path,
+            'in',
+            'Iam in external bucket',
+            'bigquery',
+            'Iam-your-external-bucket-for-trigger-test',
+        );
+
+        $tables = $testClient->listTables($idOfBucket);
+        $this->assertCount(1, $tables);
+
+        $tableIdFromExternalBucket = $tables[0]['id'];
+        $options = (new TokenCreateOptions())
+            ->addBucketPermission($this->getTestBucketId(), TokenAbstractOptions::BUCKET_PERMISSION_READ);
+        $newToken = $this->tokens->createToken($options);
+
+        // create trigger on table in external bucket should fail
+        try {
+            $this->_client->createTrigger([
+                'component' => 'orchestrator',
+                'configurationId' => 123,
+                'coolDownPeriodMinutes' => 1,
+                'runWithTokenId' => $newToken['id'],
+                'tableIds' => [
+                    $tableIdFromExternalBucket,
+                ],
+            ]);
+            $this->fail('should fail');
+        } catch (ClientException $e) {
+            $this->assertEquals('storage.triggers.triggerOnTableFromExternalBucket', $e->getStringCode());
+            $this->assertEquals(
+                sprintf('Trigger cannot be created on following tables, because they are from external buckets: %s', $tableIdFromExternalBucket),
+                $e->getMessage(),
+            );
+        }
+
+        $normalTableInStorage = $this->createTableWithRandomData('normal-table-1');
+
+        // create normal trigger
+        $trigger = $this->_client->createTrigger([
+            'component' => 'orchestrator',
+            'configurationId' => 123,
+            'coolDownPeriodMinutes' => 1,
+            'runWithTokenId' => $newToken['id'],
+            'tableIds' => [
+                $normalTableInStorage,
+            ],
+        ]);
+
+        // update existing trigger should also fail
+        try {
+            $this->_client->updateTrigger($trigger['id'], ['tableIds' => [$tableIdFromExternalBucket]]);
+            $this->fail('should fail');
+        } catch (ClientException $e) {
+            $this->assertEquals('storage.triggers.triggerOnTableFromExternalBucket', $e->getStringCode());
+            $this->assertEquals(
+                sprintf('Trigger cannot be created on following tables, because they are from external buckets: %s', $tableIdFromExternalBucket),
```

Comment:
```suggestion
                sprintf('Trigger cannot be set on following tables, because they are from external buckets: %s', $tableIdFromExternalBucket),
```